### PR TITLE
Fix broken links on contributor page readme

### PR DESCRIPTION
**Pull request by sambensim**
- Fixed 6 links as outlined in Issue #444.

- Additionally fixed a typo in the second to last paragraph (changed from "there are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here" to "there are many more ways to contribute to p5.js through non-source code contribution than ***cannot*** be exhaustively listed here").

### CODE REVIEW

**Comment by sambensim** [comment]
@all-contributors please add @sambensim for 📖 doc

File: src/content/contributor-docs/en/README.mdx [hunk 1]
```diff
@@ -18,15 +18,15 @@ This project follows the [all-contributors](https://github.com/kentcdodds/all-co
 @all-contributors please add @[your GitHub handle] for [your contribution type]
 ```
 
-You can find relevant contribution type [here](https://allcontributors.org/docs/en/emoji-key). Although we will usually automatically add you to the contributor list using the bot after merging your PR. The contributor docs are published on p5.js [website](https://p5js.org/contributor-docs/#/), and hosted on p5.js [GitHub repository](https://github.com/processing/p5.js/tree/main/contributor_docs).
+You can find relevant contribution type [here](https://allcontributors.org/docs/en/emoji-key). Although we will usually automatically add you to the contributor list using the bot after merging your PR. The contributor docs are published on p5.js [website](https://p5js.org/contribute/#/), and hosted on p5.js [GitHub repository](https://github.com/processing/p5.js/tree/main/contributor_docs).
```
Comment (limzykenneth) [inline src/content/contributor-docs/en/README.mdx:21]:
```suggestion
You can find relevant contribution type [here](https://allcontributors.org/docs/en/emoji-key). Although we will usually automatically add you to the contributor list using the bot after merging your PR. The contributor docs are published on p5.js [website](https://p5js.org/contribute/), and hosted on p5.js [GitHub repository](https://github.com/processing/p5.js/tree/main/contributor_docs).
```

File: src/content/contributor-docs/en/README.mdx [hunk 2]
```diff
@@ -49,16 +49,16 @@ For a typical contribution to the p5.js or p5.js-website repository, we will fol
...
 
 ## Non-source code contribution
 
-There are many more ways to contribute to p5.js through non-source code contribution than can be exhaustively listed here, some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc).
+There are many more ways to contribute to p5.js through non-source code contribution than cannot be exhaustively listed here, some of the ways may also involve working with some of the p5.js repositories (such as adding examples, writing tutorials for the website, etc.). Depending on what the planned contribution is, we may be able to support you in different ways so do reach out to us via any channel available to you (email, social media, [Discourse forum](https://discourse.processing.org/c/p5js/10), Discord, etc).
```
Comment (limzykenneth) [inline src/content/contributor-docs/en/README.mdx:60]: I think the original is correct here

**Review by limzykenneth — CHANGES_REQUESTED** [review]
A couple fixes in the line comments.

File: src/content/contributor-docs/en/README.mdx (hunk 2 above)
Comment (SableRaf) [reply src/content/contributor-docs/en/README.mdx]: Agreed. I can see why the original phrasing could be seen as confusing though. 

We could rephrase this in a more readable and to the point format. How about this: 

`There are many ways to contribute to p5.js beyond coding, such as adding examples, writing tutorials for the website and more. If you need help getting involved, contact us through the official [p5.js Discord](https://discord.gg/SHQ8dH25r9) or the [Processing Foundation forum](https://discourse.processing.org/c/p5js/10).`

I also clarified the CTA and added a link to Discord.

**Comment by SableRaf** [comment]
@sambensim would you mind revising this PR to focus only on the fixes to the broken links? 

Feel free to open a separate issue for the copy. Though it is not a typo, the text could be made clearer (see my inline comment).

**Comment by sambensim** [comment]
> @sambensim would you mind revising this PR to focus only on the fixes to the broken links?
> 
> Feel free to open a separate issue for the copy. Though it is not a typo, the text could be made clearer (see my inline comment).

Hey! Sorry just saw this, should I use the text you suggested earlier or just focus on the link fixes?

**Comment by SableRaf** [comment]
@sambensim No worries! Thanks for your reply :) I'd suggest focusing on the links. 

If you want to address the text please open a separate issue. It is correct as is but your intuition to correct it shows that the phrasing could be clearer.

**Comment by Qianqianye** [comment]
Thanks @sambensim for all your work on this. The [contribute](https://p5js.org/contribute/) pages on p5js.org website are built directly from p5.js library repo [contributor_docs](https://github.com/processing/p5.js/tree/main/contributor_docs) folder. So if we want to make changes, we will edit the docs in p5.js library repo [contributor_docs](https://github.com/processing/p5.js/tree/main/contributor_docs) folder,

All the broken links mentioned in this PR is fixed in this [commit](https://github.com/processing/p5.js/commit/3c86d93ee183118f534ea1471ae9be2f22423d18) in p5.js library repo, and then this [commit](https://github.com/processing/p5.js-website/commit/64b2becffcb89f2f4d3c969fee27066b0d205e8e) in p5.js website repo. In the future, if you find more broken links of contribute pages, please go ahead and open PRs within [p5.js library repo](https://github.com/processing/p5.js). 

I will close this PR for organizational purpose. Thank you!

**Pull request closed without merging**